Documentation/upgrade.md

```diff
@@ -8,12 +8,18 @@ This guide will walk you through the manual steps to upgrade the software in a R
 Rook is a distributed software system and therefore there are multiple components to individually upgrade in the sequence defined in this guide.
 After each component is upgraded, it is important to verify that the cluster returns to a healthy and fully functional state.
 
-
 This guide is just the beginning of upgrade support in Rook.
 The goal is to provide prescriptive guidance and knowledge on how to upgrade a live Rook cluster and we hope to get valuable feedback from the community that will be incorporated into an automated upgrade solution by the Rook operator.
 
 We welcome feedback and opening issues!
 
+## Supported Versions
+The supported version for this upgrade guide is **from an 0.7 release to the latest builds**. Until 0.8 is released, the latest builds are labeled such as `v0.7.1-4.g4233e03`. Build-to-build upgrades are not guaranteed to work. This guide is to test upgrades only between the official releases.
```

Comment:
right now anything starting with `v0.7.1-...` is actually from the `release-0.7 branch` since that is the only branch that has received the `v0.7.1` tag.  That last tag in master was `v0.7.0`, so any builds from master will start with `v0.7.0`, for example `v0.7.0-23.g6f2dc99`

Reply:
good catch, i'll use that tag